### ISSUE TITLE
Uchaguzi Android app - no about page

### ISSUE THREAD

**Issue by heatherleson** [ushahidi/uchaguzi#116]
The Uchaguzi Android app has no "about" page.  v 3.1.5 and not links to the website or partners.

**Comment by eyedol** [comment]
@heatherleson You mean, you don't see this?

![Screenshot_2013-03-01-13-51-58](https://f.cloud.github.com/assets/73175/209693/d9f5d54a-8277-11e2-80ce-4dc0b9fa7f82.png)

**Comment by heatherleson** [comment]
Yes, thank you

**Comment by eyedol** [comment]
@heatherleson  and you don't have this?
![Screenshot_2013-03-01-14-04-25](https://f.cloud.github.com/assets/73175/209739/a2d28a3e-8279-11e2-8fa5-9cf576534970.png)

**Comment by heatherleson** [comment]
Yes. this is what I see. I can test more tomorrow if you need me

**Comment by heatherleson** [comment]
The categories on the main site have changed so they won't sync
